test/unit/user-register.spec.js

```diff
@@ -0,0 +1,19 @@
+/* global describe, it, expect */
+
+import Vue from "vue";
+import UserRegister from "../../src/components/user-register.vue";
+
+describe("user-register.html.ep", () => {
+    it("should render correct contents", () => {
```

Comment:
I don't get why this file has four spaces. All the other code that I run "jsfmt" on has two spaces, using https://github.com/Nordaaker/convos/blob/one-point-oh/.jsfmtrc